api/src/db/migrations/versions/2025_06_27_increase_column_size_for_competition_.py

```diff
@@ -0,0 +1,42 @@
+"""Increase column size for Competition.expected_application_count
+
+Revision ID: cff668b42a9f
+Revises: 1012e0cb7c7c
+Create Date: 2025-06-27 18:18:46.862621
+
+"""
+
+import sqlalchemy as sa
```

Comment:
Linter doesn't like this - some migrations need this import, some don't - those that don't get yelled at by the inter to remove this